docs/Deploying-Pega-on-AKS.md

```diff
@@ -357,9 +357,9 @@ To customize these files, you must download them from the source github reposito
 ```
   $ helm search repo pega
   NAME                  CHART VERSION   APP VERSION     DESCRIPTION
-  pega/pega             1.4.4                           Helm chart to configure required installation and deployment configuration settings in your environment for your deployment.
-  pega/addons           1.4.4           1.0             Helm chart to configure supporting services and tools in your environment for your deployment.
-  pega/backingservices  1.4.4                           Helm Chart to provision the latest Search and Reporting Service (SRS) for your Pega Infinity deployment
+  pega/pega             2.2.0                           Helm chart to configure required installation and deployment configuration settings in your environment for your deployment.
+  pega/addons           2.2.0           1.0             Helm chart to configure supporting services and tools in your environment for your deployment.
+  pega/backingservices  2.2.0                           Helm Chart to provision the latest Search and Reporting Service (SRS) for your Pega Infinity deployment
```

Comment:
sorry but each of these updated backingservices statements in each runbook needs a period at the end. I missed that before;-/

Helm Chart to provision the latest Search and Reporting Service (SRS) for your Pega Infinity deployment"."